packages/cursorless-vscode/src/extension.ts

```diff
@@ -92,7 +92,7 @@ export async function activate(
     runIntegrationTests,
     addCommandRunnerDecorator,
     customSpokenFormGenerator,
-  } = createCursorlessEngine(
+  } = await createCursorlessEngine(
```

Comment:
Ok we don't want to wait on hat and snippets but maybe we want this one which is to wait that the cursorless engine is created? :)

Reply:
If we have no `await`s in `createCursorlessEngine` then it doesn't need to be `async`. If it's not `async` it won't return a Promise.

Reply:
Yes, as @auscompgeek points out, the function is not async on `main`, only in this PR for those two `await` statements that you added here. So no need to `await` it